src/PangenomeSchematic.js

```diff
@@ -61,25 +61,25 @@ class PangenomeSchematic extends React.Component {
         "No filepath given. Ensure chunknames in bin2file.json are correct."
       );
     console.log("Fetching", filepath);
-    return fetch(process.env.PUBLIC_URL + filepath).then((res) => res.json());
+    return fetch(filepath).then((res) => res.json());
   }
 
   loadJsonCache(url, data) {
     console.log("STEP #6: fetched chunks go into loadJsonCache");
 
-    if (data.json_version !== 16) {
+    if (data.json_version !== 17) {
```

Comment:
As Simon suggests in #86 , Can we please have some very tiny default data people can browse in the master branch of json_version === 17?

Reply:
current test_data seems outdated.

Reply:
v17 isn't ready or available yet.  We'll be releasing on 16.